### DISCUSSION TITLE
403 Forbidden on realtime request in next 13 app folder structure

### DISCUSSION THREAD

**Discussion by NixBiks** [supabase/supabase#11524] [Questions]
I'm trying to get realtime updates working in next 13 app folder structure.

I've followed [this guide ](https://supabase.com/docs/guides/auth/auth-helpers/nextjs-server-components#server-components) and can access `http://localhost:3000/realtime` just fine and see the list of posts which is fetched from the server. But the clients realtime subscribtion does not worker. See screenshot from browser console

<img width="1277" alt="Screenshot 2023-01-08 at 15 35 51" src="https://user-images.githubusercontent.com/16841792/211202396-f47eb70a-1538-43e0-8824-49120dc7bbf5.png">

As you can see I did enable realtime updates and RLS (not clear to me if RLS is a requirement too) on the table as well as a read access for all users policy

<img width="995" alt="Screenshot 2023-01-08 at 15 37 36" src="https://user-images.githubusercontent.com/16841792/211202462-860e3898-8f63-4012-878f-f78e86bfd03b.png">

<img width="1214" alt="Screenshot 2023-01-08 at 15 44 06" src="https://user-images.githubusercontent.com/16841792/211202602-ef772f07-2ad3-417b-a267-9cd349b27c76.png">

<img width="970" alt="Screenshot 2023-01-08 at 20 00 02" src="https://user-images.githubusercontent.com/16841792/211213939-e9c0c1b0-2958-4e2e-aee3-7d92e72bd81e.png">

I'm logged in using magic email link (using the inbucket service). In `middleware.ts` I'm redirected if not logged in...

What am I missing here?

### Edit

I guess it's simply because [real-time doesn't work local currently](https://github.com/supabase/realtime/issues/415)

**Comment by w3b6x9** [comment]
@NixBiks https://github.com/supabase/realtime/issues/415 was resolved so it should be working locally now. Can you try it again?
